app/gui2/src/components/widgets/DropdownWidget.vue

```diff
@@ -108,7 +108,7 @@ export interface DropdownEntry {
   margin-top: calc(0px - var(--dropdown-extend));
   padding-top: var(--dropdown-extend);
   background-color: var(--dropdown-bg);
-  border-radius: 16px;
+  border-radius: 19px;
```

Comment:
It would be nice if these radii were `calc`ed, so that if padding is changed they are still correct.